resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@sbsmith86 ahh I thiiink I see what you're saying. I'm going to try doing it that way and we'll find out for sure